content/applications/hr.rst

```diff
@@ -11,4 +11,5 @@ Human resources
    hr/fleet
    hr/payroll
    hr/recruitment
+   hr/appraisals
```

Comment:
for the structure, I think it'd make more sense to leave Recruitment and Referrals together, while Appraisals could make more sense close to employees.

Reply:
Just wanted to let you know this was changed as requested, and this has been reorganized.